homeassistant/components/sensor/ebusd.py

```diff
@@ -0,0 +1,185 @@
+from datetime import timedelta
+from datetime import datetime
+import subprocess
+import logging
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_NAME, CONF_MONITORED_VARIABLES, STATE_ON, STATE_OFF, STATE_UNKNOWN)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'ebusd'
+CONF_CIRCUIT = 'circuit'
+CONF_TIME = 0
+
+BASE_URL = 'ebusctl read -m {2} -c {0} {1}'
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=15)
+SCAN_INTERVAL = timedelta(seconds=30)
+
+SENSOR_TYPES = {
+    'hc1ActualFlowTempDesired': ['Hc1ActualFlowTempDesired','°C', 'mdi:thermometer', 'decimal'],
```

Comment:
The sensor name should be somehow human-readable. `'Hc1ActualFlowTempDesired'` -> `'actual_flow_temperature'` or `'z1NightTemp'` -> `'night_temp'`. users should care about  the prefixes (Hc1, hwc, z1).